src/pages/rewards/[address]/index.tsx

```diff
@@ -225,10 +289,64 @@ const Rewards: React.FC = () => {
                 void notify(farm, toWei(amount));
                 refresh();
               }}
-              mr={2}
+              mr={1}
             >
               Notify custom {lookup.rewardTokenSymbol}
             </Button>
+
+            <Button
```

Comment:
Would be good to check if the connected wallet is the owner of this farm. Only the `owner` is allowed to set the `rewardsDistribution`